Description: of all MIPS architectures, JIT should be only enabled on mipsel
Author: Dmitry Shachnev <mitya57@debian.org>
Forwarded: not-yet
Last-Update: 2017-07-05

--- a/src/qml/jsruntime/qv4global_p.h
+++ b/src/qml/jsruntime/qv4global_p.h
@@ -107,7 +107,7 @@
 #  if defined(Q_OS_LINUX)
 #    define V4_ENABLE_JIT
 #  endif
-#elif defined(Q_PROCESSOR_MIPS_32) && defined(Q_OS_LINUX)
+#elif defined(Q_PROCESSOR_MIPS_32) && defined(Q_OS_LINUX) && Q_BYTE_ORDER == Q_LITTLE_ENDIAN && _MIPS_SIM == _ABIO32
 #  define V4_ENABLE_JIT
 #endif
 
